lib/MusicBrainz/Server/Controller/Role/Collection.pm

```diff
@@ -33,28 +33,38 @@ role
 
     method _all_collections => sub {
         my ($self, $c) = @_;
-        return [ $c->model('Collection')->find_all_by_entity($entity_type, $c->stash->{$entity_name}->id) ];
+        my ($collections) = $c->model('Collection')->find_by({
+            entity_id => $c->stash->{$entity_name}->id,
+            entity_type => $entity_type,
+        });
```

Comment:
Does the new one (`find_by` with `entity_id` and `entity_type` specified) produce the same output as `find_all_by_entity`?